include/vm_phys.h

```diff
@@ -0,0 +1,38 @@
+#ifndef _VM_PHYS_H_
+#define	_VM_PHYS_H_
+
+#include <queue.h>
+#include <common.h>
+
+#define VM_NFREEORDER 9
+#define VM_RESERVED 1
+
+typedef uintptr_t vm_addr_t;
+typedef uintptr_t vm_paddr_t;
+typedef struct vm_page* vm_page_t;
```

Comment:
I understand your point.

There is one curiosity however. In FreeBSD's code vm_page_t is defined as **pointer** to page.